src/components/monitor/HeroSection.tsx

```diff
@@ -0,0 +1,22 @@
+'use client';
```

Comment:
Why? We are not using nextJS 14, where all components are SSR by default. Do we need use client for this version?